docs/input-kafka.asciidoc

```diff
@@ -71,7 +71,9 @@ inserted into your original event, you'll have to use the `mutate` filter to man
 
 This plugin supports these configuration options plus the <<plugins-{type}s-{plugin}-common-options>> described later.
 
-NOTE: Some of these options map to a Kafka option. See the https://kafka.apache.org/documentation for more details.
+NOTE: Some of these options map to a Kafka option. Defaults usually reflect the Kafka default setting.
```

Comment:
@karenzone could you please take a look if this block sounds meaningful, 
was trying to state that the new defaults might change as Kafka's underlying defaults change : 
> our defaults will attempt to follow sane defaults and best-practices for these settings from the kafka consumer.

Reply:
@kares. I like what you have. Consider adding "and might change if Kafka's defaults change" at the end of the note to make the user impact obvious. 

Like this: 
"NOTE: Some of these options map to a Kafka option. Defaults usually reflect the Kafka default setting, and might change if Kafka's defaults change."